ddtrace/appsec/__init__.py

```diff
@@ -0,0 +1,11 @@
+from ddtrace.appsec.management import Management
+
+
+mgmt = Management()
```

Comment:
```suggestion
_mgmt = Management()
```

maybe worth being certain to point out that this is internal 🙂